bcda/models/models_test.go

```diff
@@ -79,6 +81,38 @@ func (s *ModelsTestSuite) TestCreateACO() {
 	assert.NotNil(err)
 }
 
+func (s *ModelsTestSuite) TestACOPublicKeySave() {
+	assert := s.Assert()
+
+	// Setup ACO
+	cmsID := "A9994"
+	var aco ACO
+	s.db.Find(&aco, "aco_id = ?", cmsID)
```

Comment:
This should also be First, Take or Last, all of which return a single record, not a list of all records (which may happen to be one record)

Reply:
On reading the rest of this I don't really see the benefit of looking up an existing ACO in the DB for this test.  Nothing about the state of the existing ACO is relevant to the test and it would just be cleaner to make a new one for this test, and delete it when the test ends.  That would simplify the test setup and ensure that there are no side effects like some other test loading up this ACO while it has the "wrong" key attached to it.

Reply:
OK, going with `First` and a newly created ACO.